addons/hashes/fnc_isHash.sqf

```diff
@@ -23,4 +23,4 @@ SCRIPT(isHash);
 // -----------------------------------------------------------------------------
 params ["_hash"];
 
-_hash isEqualType [] && {count _hash == 4} && {(_hash select HASH_ID) isEqualTo TYPE_HASH}
+_hash isEqualType [] && {count _hash == 5} && {(_hash select HASH_ID) isEqualTo TYPE_HASH}
```

Comment:
This means isHash will report false for hashes stored in the profile namespace in a previous version.